lua/nvim-tree/git/runner.lua

```diff
@@ -6,8 +6,27 @@ Runner.__index = Runner
 
 function Runner:_parse_status_output(line)
   local status = line:sub(1, 2)
+  local path = line:sub(4, -2)
+  if utils.str_find(status, "R") then
+    -- rename have format of "from -> to"
+    -- we just what the "to" part
+    if path:sub(1, 1) == '"' then
+      -- incase " -> " is a part of the file name
```

Comment:
That feels awfully user case specific.

Reply:
What do you mean by use case specific? If we see the first character as `"` then the file names are definitely enclosed in quotes no? And if that's the case there might be ` -> ` present in the file name so the only way to reliably tell which one is ` -> `  is to search for `" -> "`?

Reply:
Apologies, I took it out of context completely. Of course this is a rename marker here. Sorry to have wasted your time.